ui/js/graphs.js

```diff
@@ -738,6 +741,24 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
   var testArray = [];
   var series = [];
   $scope.addTestData = function () {
+    if ($scope.addedTestList.length > 6) {
+      var a = window.confirm('WARNING: You are about to load ' + $scope.addedTestList.length +
+                           ' tests to the graph.\n' +
+                           'Loading more than 6 is not recommended. Do it anyway?');
+      if (a == true) {
+        addTestToGraph();
+      }
+    } else if ($scope.addedTestList.length + testCounter > 6) {
+      var a = window.confirm('WARNING: You have ' + testCounter + ' tests in the graph already' +
+                          ' and You are about to load ' + $scope.addedTestList.length + ' more test.\n'
+                           + 'Loading more than 6 is not recommended. Do it anyway?');
+      if (a == true) {
+        addTestToGraph();
+      }
+    } else addTestToGraph();
```

Comment:
Is that mean we don't need to consider how much tests been putted in graph already? We just need to tell user don't put more than 6 tests at once.